src/components/PromptForm.svelte

```diff
@@ -87,7 +87,7 @@
 			required
 			spellcheck="false"
```

Comment:
```suggestion
```

I support removing this @edwardsharp. I believe I added it because spellcheck can often trigger on technical language and the like, but I hear you that it is good to have the squiggles when you need them.